integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForDeleteCommand.scala

```diff
@@ -110,6 +112,18 @@ private[sql] case class CarbonProjectForDeleteCommand(
       HorizontalCompaction.tryHorizontalCompaction(sparkSession, carbonTable,
         isUpdateOperation = false)
 
+      if (CarbonProperties.getInstance().isDistributedPruningEnabled(carbonTable
```

Comment:
Can u please move this code to some utility

Reply:
Moved to  DeleteExecution.scala